### ISSUE TITLE
Syncing with OneDrive have problem when the filename contains fullwidth symbols.

### ISSUE THREAD

**Issue by maple3142** [rclone/rclone#3378]
#### What is the problem you are having with rclone?

Syncing with OneDrive have problem when the filename contains fullwidth symbols.

#### What is your rclone version (output from `rclone version`)

v1.48.0

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)

Windows 10, 64bit

####  Which cloud storage system are you using? (eg Google Drive)

OneDrive(Business)

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

The `D:\test` folder contains an empty text file called `～.txt`. (PS: **～** is not **~**, the former is fullwidth)

```bash
rclone sync test OD5T:/test # This will create a `～.txt` in `OD5T:/test`
rclone check test OD5T:/test # Have problem comparing the files
rclone sync test OD5T:/test # This will delete `～.txt` from `OD5T:/test`
```

#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

```bash
D:\>rclone sync test OD5T:/test -vv
2019/07/26 20:47:02 DEBUG : rclone: Version "v1.48.0" starting with parameters ["rclone" "sync" "test" "OD5T:/test" "-vv"]
2019/07/26 20:47:02 DEBUG : Using config file from "C:\\Users\\Administrator\\.config\\rclone\\rclone.conf"
2019/07/26 20:47:05 INFO  : One drive root 'test': Waiting for checks to finish
2019/07/26 20:47:05 INFO  : One drive root 'test': Waiting for transfers to finish
2019/07/26 20:47:05 DEBUG : ～.txt: Starting singlepart upload
2019/07/26 20:47:06 INFO  : ～.txt: Copied (new)
2019/07/26 20:47:06 INFO  : Waiting for deletions to finish
2019/07/26 20:47:06 INFO  :
Transferred:             0 / 0 Bytes, -, 0 Bytes/s, ETA -
Errors:                 0
Checks:                 0 / 0, -
Transferred:            1 / 1, 100%
Elapsed time:        4.8s

2019/07/26 20:47:06 DEBUG : 6 go routines active
2019/07/26 20:47:06 DEBUG : rclone: Version "v1.48.0" finishing with parameters ["rclone" "sync" "test" "OD5T:/test" "-vv"]
```

```bash
D:\>rclone check test OD5T:/test -vv
2019/07/26 20:47:31 DEBUG : rclone: Version "v1.48.0" starting with parameters ["rclone" "check" "test" "OD5T:/test" "-vv"]
2019/07/26 20:47:31 DEBUG : Using config file from "C:\\Users\\Administrator\\.config\\rclone\\rclone.conf"
2019/07/26 20:47:31 INFO  : One drive root 'test': Waiting for checks to finish
2019/07/26 20:47:32 ERROR : ～.txt: File not in One drive root 'test'
2019/07/26 20:47:32 ERROR : ~.txt: File not in Local file system at \\?\D:\test
2019/07/26 20:47:32 NOTICE: One drive root 'test': 1 files missing
2019/07/26 20:47:32 NOTICE: Local file system at \\?\D:\test: 1 files missing
2019/07/26 20:47:32 NOTICE: One drive root 'test': 2 differences found
2019/07/26 20:47:32 Failed to check with 3 errors: last error was: 2 differences found
```

```bash
D:\>rclone sync test OD5T:/test -vv
2019/07/26 20:50:22 DEBUG : rclone: Version "v1.48.0" starting with parameters ["rclone" "sync" "test" "OD5T:/test" "-vv"]
2019/07/26 20:50:22 DEBUG : Using config file from "C:\\Users\\Administrator\\.config\\rclone\\rclone.conf"
2019/07/26 20:50:23 INFO  : One drive root 'test': Waiting for checks to finish
2019/07/26 20:50:23 INFO  : One drive root 'test': Waiting for transfers to finish
2019/07/26 20:50:23 DEBUG : ～.txt: Starting singlepart upload
2019/07/26 20:50:24 INFO  : ～.txt: Copied (new)
2019/07/26 20:50:24 INFO  : Waiting for deletions to finish
2019/07/26 20:50:24 INFO  : ~.txt: Deleted
2019/07/26 20:50:24 INFO  :
Transferred:             0 / 0 Bytes, -, 0 Bytes/s, ETA -
Errors:                 0
Checks:                 1 / 1, 100%
Transferred:            1 / 1, 100%
Elapsed time:        2.2s

2019/07/26 20:50:24 DEBUG : 6 go routines active
2019/07/26 20:50:24 DEBUG : rclone: Version "v1.48.0" finishing with parameters ["rclone" "sync" "test" "OD5T:/test" "-vv"]
```

**Comment by ncw** [comment]
This will be fixed by #3148 when it lands

**Comment by ncw** [comment]
This should be fixed with v1.50
